features/step_definitions/security/generic_steps.rb

```diff
@@ -1,5 +1,5 @@
+# frozen_string_literal: true
+
 Then /^I should be denied the access$/ do
-  # assert has_content?('Access Denied')
-  assert_equal 403, page.status_code
+  assert_content 'Access Denied'
```

Comment:
Also could you explain this?

Reply:
Selenium does not support `status_code`: https://github.com/seleniumhq/selenium-google-code-issue-archive/issues/141